packages/rocketchat-ui/client/views/app/popover.js

```diff
@@ -129,5 +127,84 @@ Template.popover.events({
 		}
 
 		popover.close();
+	},
+	'click [data-type="sidebar-item"]'(e, instance) {
+		popover.close();
+		const { rid, name, template } = instance.data.data;
+
+		if (e.currentTarget.dataset.id === 'hide') {
+			let warnText;	
+			switch (template) {
+				case 'c': warnText = 'Hide_Room_Warning'; break;
+				case 'p': warnText = 'Hide_Group_Warning'; break;
+				case 'd': warnText = 'Hide_Private_Warning'; break;
+			}
```

Comment:
What are we doing about livechat?  I have 2 rooms stuck open on mine that I can't hide or get rid of :joy:

Reply:
I'll add a warning text for livechat :)

Reply:
\o/ awesome 🙂